src/main/java/components/SavedDataLoader.java

```diff
@@ -0,0 +1,62 @@
+package components;
+
+import domain.AttendanceBook;
+import domain.AttendanceHistory;
+import domain.Crew;
+import domain.Crews;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+import java.util.stream.Collectors;
+import utils.FileReader;
+
+public class SavedDataLoader {
+    private static final String FILE_PATH = "./src/main/resources/attendances.csv";
+    private static final int CREW = 0;
+    private static final int ATTENDANCE = 1;
+    private static final String DELIMITER = ",";
+    private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm");
+    private final List<String[]> loadedData;
```

Comment:
사소한 부분일 수 있겠지만, 상수와 상태는 줄바꿈을 통해서 가독성을 높여주면 좋을 것 같아요.
```suggestion
    private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm");
    
    private final List<String[]> loadedData;
```